src/router.js

```diff
@@ -28,3 +38,21 @@ const router = new Router({
 })
 
 export default router
+
+// check if feature is allowed and redirect if not
+async function featureAvailable(apollo, networkId, to) {
+  const {
+    data: { networks }
+  } = await apollo.query({
+    query: NetworkCapabilities(networkId)
```

Comment:
It might be possible to query for the feature directly in the GraphQL. Effectively turning the result into a boolean. 

Just thought of this now, but I need to google the syntax.